String.cxx

```diff
@@ -77,13 +75,9 @@ void String::setCharAt(unsigned index, const char* c) { m_u[index] = c; }
 
 void String::toCharArray(const char* buffer[], unsigned length) noexcept {
```

Comment:
Wrong signature; should be `void String::toCharArray(char* buffer, unsigned length)`